diff --git a/node_modules/@medusajs/core-flows/dist/cart/workflows/get-variants-and-items-with-prices.js b/node_modules/@medusajs/core-flows/dist/cart/workflows/get-variants-and-items-with-prices.js
--- a/node_modules/@medusajs/core-flows/dist/cart/workflows/get-variants-and-items-with-prices.js
+++ b/node_modules/@medusajs/core-flows/dist/cart/workflows/get-variants-and-items-with-prices.js
@@ -103,9 +103,9 @@ exports.getVariantsAndItemsWithPrices = (0, workflows_sdk_1.createWorkflow)(expo
                 isCustomPrice: isCustomPrice,
             };
             if (variant && !isCustomPrice) {
-                input.unitPrice = calculatedPriceSet.calculated_amount;
+                input.unitPrice = calculatedPriceSet?.calculated_amount;
                 input.isTaxInclusive =
-                    calculatedPriceSet.is_calculated_price_tax_inclusive;
+                    calculatedPriceSet?.is_calculated_price_tax_inclusive;
             }
             const preparedItem = (0, prepare_line_item_data_1.prepareLineItemData)(input);
             return {
